cord/http/utils.py

```diff
@@ -91,3 +93,56 @@ def upload_to_signed_url_list(file_paths, signed_urls, querier, orm_class):
     return data_uid_list
 
 
+async def upload_to_signed_url_list_async(file_paths, signed_urls, querier, orm_class):
```

Comment:
Making those functions async is an interesting choice. Why? Easy parallel upload?

Reply:
Yes, I personally prefer the non-blocking paradigm. The alternative would be to spawn a bunch of threads to do the same. I think it is a matter of preference but I feel like the industry is trying to push towards the non-blocking paradigm.